autopxd/__init__.py

```diff
@@ -303,9 +303,11 @@ def lines(self):
 
 
 def preprocess(code, extra_cpp_args=[]):
-    proc = subprocess.Popen([
-        'cpp', '-nostdinc', '-D__attribute__(x)=', '-I', BUILTIN_HEADERS_DIR,
-    ] + extra_cpp_args + ['-'], stdin=subprocess.PIPE, stdout=subprocess.PIPE)
+    proc = subprocess.Popen(
+        ['cpp', '-nostdinc', '-D__attribute__(x)=',
```

Comment:
I exceeded pep8 max line length with the extra argument. Please suggest the style you prefer.